usage/usage.go

```diff
@@ -1,31 +1,49 @@
-/*
-Copyright 2023 The OpenEBS Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-	http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
 package usage
 
 import (
+	"encoding/base64"
+	"os"
 	"strconv"
 
+	k8sapi "github.com/openebs/lib-csi/pkg/client/k8s"
 	"k8s.io/klog/v2"
 
 	ga4Client "github.com/openebs/google-analytics-4/pkg/client"
 	ga4Event "github.com/openebs/google-analytics-4/pkg/event"
-	k8sapi "github.com/openebs/lib-csi/pkg/client/k8s"
 )
 
+// apiCreds reads envs and decodes base64 input for GA-4 MeasurementId and ApiSecret.
+// Returns defaults if envs are unset/invalid.
+func apiCreds() (string, string) {
+	encodedId, okId := os.LookupEnv(MeasurementIdEnv)
```

Comment:
Can we call okId to something like, idExists, secretExists?

Reply:
Sure, why not.